task-2/.eslintrc.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
```

Comment:
There's no need to have 2 ESLint configuration files.
You currently have `.eslintrc.js` and `.eslintrc.json` with different configurations.
Consider sticking to just one config file format and combining all the configs there.

Reply:
Eslint configurations have now been complied into one config file - ( .eslint.js ).